resources/authgear/templates/en/translation.json

```diff
@@ -1006,6 +1006,7 @@
   "v2.error.screen-title": "Unexpected Issues",
   "v2.error.server": "Server encountered an internal error.",
   "v2.error.settings-account-conflict": "{IdentityTypeIncoming, select, login_id {This {LoginIDNameIncoming} is already registered.} oauth {This {OAuthProviderNameIncoming} account is already registered.} other {This account is already registered.}} Please use another one.",
+  "v2.error.settings-token-invalid": "<p>This page is invalid or has expired.</p><p>Please return and retry your request.</p>",
```

Comment:
Why do we need to wrap it in `<p>`? We seldom do this.

Reply:
Waiting for new copywriting so copied from InvalidWebappSession for now.